src/components/notes/editor.jsx

```diff
@@ -42,10 +42,7 @@ export default function Editor(props) {
 
   // plugins and config
   const isPublic = useContext(IsPublicContext)
-  const { userName, userId } = useUser({
-    userName: props.userName,
-    cozyClient
-  })
+  const { userName, userId } = useUser()
```

Comment:
Hmm, `props.userName` was never set?

Reply:
For me Editor is only used into 2 places : 
- https://github.com/cozy/cozy-notes/blob/3b2fd1e6f109a3e262e6d6de179c039d546f0887/src/components/app.jsx#L39
- https://github.com/cozy/cozy-notes/blob/3b2fd1e6f109a3e262e6d6de179c039d546f0887/src/components/app.jsx#L105
No of them shared an userName property but I can be wrong